### ISSUE TITLE
New ANT2 music

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#411]
Alien-like music for the game. Fully ready and loopable:

http://team.secretchronicles.de/~quintus/stuff/ANT2.ogg

This music is originally by Ashburry ([forum posting](http://forum.secretchronicles.de/topics/212/posts/1019), and has been modified by @sydneyjd to loop properly. License is CC-BY 4.0 as it was posted on the forum and @sydneyjd generally licenses under CC-BY according to [his own postings in the forum](http://forum.secretchronicles.de/topics/227/posts/1083).

Valete,
Quintus

**Comment by datahead8888** [comment]
In what release will we probably add the Ant2 music and Nightfall music (#412) by Ashburry?

**Comment by Quintus** [comment]
datahead8888 <notifications@github.com> writes:

> In what release will we probably add the Ant2 music

The song has a pause at the end, which must be removed before it can be
included. Otherwise, it can directly be included.

> and Nightfall music (#412) by Ashburry?

Nightfall is no original artwork for TSC and had some discussion about
whether it fits the game. Please continue the discussion about that
music in #412.

Vale,
Quintus

-- 
#!/sbin/quintus
Blog: http://www.guelkerdev.de

GnuPG key: F1D8799FBCC8BC4F

**Comment by sydneyjd** [comment]
The song has a pause on the end? I just checked it out with Audacity and it most certianly does not. It does fade out and fade it, but thats what Ashbury did. 
To me this song is ready to be implemented.

**Comment by datahead8888** [comment]
Yeah, I could find any major pause last night either, at the end of the song.  It fades a little at the end, but I think some games do this.

**Comment by Bugsbane** [comment]
Overall, I really like it. I feel that the part at the very start (up to the 21 second mark) is stylistically out of place of the rest of the game by being overly minimal and dark, but everything after that I feel would fit, as long as we have it looping effectively.

**Comment by datahead8888** [comment]
In what release will we add this?

**Comment by Quintus** [comment]
I have problems with this song fitting the style of TSC. It is certainly good, but I think it is more of a sci-fi game song than one for TSC. I'll not stand in the way when including this, though. I just wanted to raise my concerns.

Vale,
Quintus
